### ISSUE TITLE
Error not bubble up

### ISSUE THREAD

**Issue by JTravis76** [lukemurray/EntityGraphQL#43]
While leaping :) from v0.21.0 to 0.60.0, ran into this.

GraphiQL started throwing this error:
![image](https://user-images.githubusercontent.com/9905282/81004398-5b489a00-8e1a-11ea-892c-47ee4b314288.png)

Even updated to 0.17.5 with same message.

Decided to remove the nuget package and reference the project directly. Then VS threw this message.
![image](https://user-images.githubusercontent.com/9905282/81006131-fcd0eb00-8e1c-11ea-93ac-7572963e1223.png)

Added the custom scalar type and the **`Failed to fetch`** error when away.
```cs
schema.AddCustomScalarType(typeof(DateTime), "DateTime");
schema.AddCustomScalarType(typeof(DateTime?), "DateTime");
```

Not sure why it didn't bubble up into the `QueryResult`.

**Comment by lukemurray** [comment]
Hmm, I can't seem to reproduce, can you reproduce in the demo project by changing it?

**Comment by JTravis76** [comment]
Attempted in the demo project, but much trouble has I am using Core 3.1 and not 2.2.

I thinking it would be the same issue if you commented out your custom scalar types. Then tried running it.

```cs
 public class GraphQLSchema
    {
        public static SchemaProvider<DemoContext> MakeSchema()
        {
            // build our schema directly from the DB Context
            var demoSchema = SchemaBuilder.FromObject<DemoContext>();

            //demoSchema.AddCustomScalarType(typeof(DateTime), "Date");
            //demoSchema.AddCustomScalarType(typeof(DateTime?), "Date");
```

If that doesn't throw the error, I will attempt again tomorrow.

**Comment by lukemurray** [comment]
That is what I tried. It worked find. I shall have dot net 3.1. It happily runs the 2.2 project. Although I will upgrade the demo project when I next get the chance

________________________________

**Comment by JTravis76** [comment]
Another thought. When using GraphiQL, which schema file are you loading; SDL or Introspection? 

I notice your demo wanting to create a SDL.

The error generated from the Introspection query.

**Comment by lukemurray** [comment]
Yep, I just ran the demo project which launches Grahiql and that runs the
introspection. I didn't get any error even when commenting out the dates

**Comment by JTravis76** [comment]
Couldn't get the demo to work with 3.1.1, so I installed v 2.2.207 to test out the demo. Commented out the two custom scalar methods, then run.

VS did break with error "No mapped entry found...", however the error was different in GraphiQL.  
Confirmed and using the same version 0.17.5.

![image](https://user-images.githubusercontent.com/9905282/82483900-23db1e00-9aa7-11ea-82a2-41926ce9bc8d.png)

Even when referencing the library via nuget. The same error... without the VS break.  
Probably not a huge deal, usually I see a introspection error instead.
